### PR TITLE
multi_disk_install: add parameters for system disk

### PR DESCRIPTION
**Pull request by YongxueHong**
Add two parameters for comfirming the system disk

Signed-off-by: Yongxue Hong <yhong@redhat.com>

ID: 1489761

### CODE REVIEW

**Comment by Zhengtong** [comment]
LGTM 
acked-by: Zhengtong Liu <zhengtli@redhat.com>

File: generic/tests/cfg/unattended_install.cfg [hunk 1]
```diff
@@ -51,6 +51,9 @@
         - multi_disk_install:
             no ide
             only unattended_install..cdrom
+            serial_name = SYSTEM_DISK0
+            blk_extra_params_image1 = "serial=${serial_name}"
+            cmd_only_use_disk = "ignoredisk --only-use=disk/by-id/*${serial_name}"
```
Comment (CongLi) [inline generic/tests/cfg/unattended_install.cfg]: could you show me where use this cmd 'cmd_only_use_disk', I did not find it.

**Comment by CongLi** [comment]
Based on: https://github.com/avocado-framework/avocado-vt/pull/1156

**Comment by YongxueHong** [comment]
@CongLi  Sorry for that. Thanks for your comment.

**Comment by qunfang** [comment]
Acked.

00:14:39 INFO | PASS 1-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.ppc64le.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native

01:16:56 INFO | PASS 2-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.ppc64le.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_threads

**Comment by CongLi** [comment]
Ack.

**Pull request merged**